requirements/stl.txt

```diff
@@ -0,0 +1,3 @@
+# stl
+
+numpy-stl
```

Comment:
one other thing that needs to happen is in `setup.py`, take a look at how gdspy is handled there and it should be straightforward. This tells pip to include `stl`  as a requirement.